tests/locationConfigTests.json

```diff
@@ -5,7 +5,12 @@
         "legacyAwsBehavior": true,
         "details": {}
     },
-    "file-test": {
+    "scality-internal-file": {
+        "type": "file",
+        "legacyAwsBehavior": false,
+        "details": {}
+    },
+    "file": {
```

Comment:
I remember the collision between this ('file' as location name) and implName (sometimes 'file') caused some undetected bugs and we tried to rename the location names so there would be no such collisions, is there a reason to go back?

In any case since these prs are crucial for the rcs/ga I think we can let it slide but make a note to change these back to different names afterwards if there isn't a reason we are sticking to just 'file' or 'mem' for the location names.

Reply:
What is the difference between "scality-internal-file" and "file"? And "scality-internal-mem" and "mem"? It looks like we are usually using just the "scality-internal" ones for tests, are we using the "file" and "mem" locations for anything?

Reply:
@electrachong, you are right. My fault. @nicolas2bert, will you please check if we can get rid of those added locations?